src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -29,16 +29,19 @@
 examples:
 -   name: Create a self-signed certificate with the default policy and add it to a virtual machine.
     text: |
-        az keyvault certificate create --vault-name vaultname -n cert1 \
+        az keyvault certificate create --vault-name vaultname -n cert1 \\
           -p "$(az keyvault certificate get-default-policy)"
 
-        secrets=$(az keyvault secret list-versions --vault-name vaultname \
+        secrets=$(az keyvault secret list-versions --vault-name vaultname \\
           -n cert1 --query "[?attributes.enabled].id" -o tsv)
 
         vm_secrets=$(az vm secret format -s "$secrets")
 
-        az vm create -g group-name -n vm-name --admin-username deploy  \
+        az vm create -g group-name -n vm-name --admin-username deploy  \\
           --image debian --secrets "$vm_secrets"
+-   name: Create a Key Vault certificate.
+    text: az keyvault certificate create --disabled false --policy {policy} --vault-name MyVault --name MyCertificate
+    crafted: true
```

Comment:
Policy is a complex JSON string. A placeholder is not appropriate. This should either have a real value or some other valid syntax like `@policy.json` or  `$(az keyvault certificate get-default-policy)`